{
  "chaoslib-version": "1.21.0",
  "platform": "Windows-10-10.0.19043-SP0",
  "node": "LNAR-5CD9051V6X",
  "experiment": {
    "title": "Moving a file from under our feet is forgivable",
    "description": "Our application should re-create a file that was removed",
    "contributions": {
      "reliability": "high",
      "availability": "high"
    },
    "controls": [
      {
        "type": "probe",
        "name": "tracing",
        "scope": "before",
        "automatic": true,
        "provider": {
          "type": "python",
          "module": "chaostracing.control"
        }
      }
    ],
    "steady-state-hypothesis": {
      "title": "Validate Steady state",
      "probes": [
        {
          "name": "test-steady-state",
          "type": "probe",
          "tolerance": 15,
          "provider": {
            "type": "python",
            "module": "recursionVisualizer",
            "func": "steadystate",
            "arguments": {
              "i": 15
            }
          }
        }
      ]
    },
    "method": [
      {
        "name": "test-readconfig-module",
        "type": "action",
        "provider": {
          "type": "python",
          "module": "turtletest",
          "func": "readconfig",
          "arguments": {
            "cfgdict": "default.cfg"
          }
        }
      },
      {
        "name": "test-configdict-module",
        "type": "action",
        "provider": {
          "type": "python",
          "module": "turtletest",
          "func": "config_dict",
          "arguments": {
            "filename": "test.py"
          }
        }
      },
      {
        "name": "test-onscreenclick-module",
        "type": "action",
        "provider": {
          "type": "python",
          "module": "turtletest",
          "func": "onscreenclick",
          "arguments": {
            "fun": "None"
          }
        }
      },
      {
        "name": "test-ondrag-module",
        "type": "action",
        "provider": {
          "type": "python",
          "module": "turtletest",
          "func": "ondrag",
          "arguments": {
            "fun": "None"
          }
        }
      },
      {
        "type": "action",
        "name": "test-turtle-forward-module",
        "provider": {
          "type": "python",
          "module": "turtle",
          "func": "forward",
          "arguments": {
            "distance": 34
          }
        },
        "pauses": {
          "after": 10
        }
      },
      {
        "type": "action",
        "name": "test-turtle-backward-module",
        "provider": {
          "type": "python",
          "module": "turtle",
          "func": "backward",
          "arguments": {
            "distance": 56
          }
        },
        "pauses": {
          "after": 10
        }
      },
      {
        "type": "action",
        "name": "test-turtle-left",
        "provider": {
          "type": "python",
          "module": "turtle",
          "func": "left",
          "arguments": {
            "angle": 45
          }
        },
        "pauses": {
          "after": 10
        }
      },
      {
        "type": "action",
        "name": "test-turtle-rt-module",
        "provider": {
          "type": "python",
          "module": "turtle",
          "func": "right",
          "arguments": {
            "angle": 45
          }
        },
        "pauses": {
          "after": 10
        }
      },
      {
        "ref": "test-steady-state"
      }
    ],
    "dry": false
  },
  "start": "2021-11-22T10:44:22.339971",
  "status": "failed",
  "deviated": false,
  "steady_states": {
    "before": {
      "steady_state_met": false,
      "probes": [
        {
          "activity": {
            "name": "test-steady-state",
            "type": "probe",
            "tolerance": 15,
            "provider": {
              "type": "python",
              "module": "recursionVisualizer",
              "func": "steadystate",
              "arguments": {
                "i": 15
              }
            }
          },
          "output": null,
          "status": "failed",
          "exception": [
            "Traceback (most recent call last):\n",
            "  File \"C:\\Users\\asunder\\.venvs\\chaostk\\lib\\site-packages\\chaoslib\\provider\\python.py\", line 55, in run_python_activity\n    return func(**arguments)\n",
            "  File \"C:\\Users\\asunder\\Python\\Recursion Visulaizer\\chaostest\\recursionVisualizer.py\", line 30, in steadystate\n    t.down(i)\n",
            "TypeError: pendown() takes 1 positional argument but 2 were given\n",
            "\nDuring handling of the above exception, another exception occurred:\n\n",
            "chaoslib.exceptions.ActivityFailed: TypeError: pendown() takes 1 positional argument but 2 were given\n"
          ],
          "start": "2021-11-22T10:44:22.345117",
          "end": "2021-11-22T10:44:22.346138",
          "duration": 0.001021,
          "tolerance_met": false
        }
      ]
    },
    "after": null,
    "during": []
  },
  "run": [],
  "rollbacks": [],
  "end": "2021-11-22T10:44:22.347151",
  "duration": 0.007180452346801758
}